import "IpcMessage.proto";
import "User.proto";

package METRO.CRO.MESSAGES;

message ClassroomInfo {
	extend IpcMessage {
		optional ClassroomInfo message = 203;
	}

	required string u_id = 1;
	required string cate_id = 2;
	required string name = 3;

	required User teacher = 4;

	optional string description = 5;
	optional string imgurl = 6;
	optional string time_open = 7;
	optional string time_close = 8;
}